### PR TITLE
samples: cellular: http_update: app: update binaries and add for nRF9161

### PR DESCRIPTION
**Pull request by eivindj-nordic**
Update binaries in AWS and add separate binaries for nRF9161 as the DK have different pin configuration.

### CODE REVIEW

**Comment by eivindj-nordic** [comment]
Removing `changelog-entry-required` label as this comes under the support for nRF9161 in cellular samples and there is already an changelog entry.

File: samples/cellular/http_update/application_update/boards/nrf9160dk_nrf9160_ns.conf [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2023 Nordic Semiconductor ASA
```
Comment (anhmolt) [inline samples/cellular/http_update/application_update/boards/nrf9160dk_nrf9160_ns.conf:1]: Missing a line with `#` at the top of the file. Same for `boards/nrf9161dk_nrf9161_ns.conf`.

Apart from that, looks good.

**Review by anhmolt — APPROVED** [review]

File: samples/cellular/http_update/application_update/boards/nrf9160dk_nrf9160_ns.conf (hunk 1 above)
Comment (eivindj-nordic) [reply samples/cellular/http_update/application_update/boards/nrf9160dk_nrf9160_ns.conf:2]: updated

**Comment by eivindj-nordic** [comment]
The `test-fw-nrfconnect-nrf-iot_samples` plan is failing due to a failure with the ciphersuite sample. This is unrelated to this PR.

**Review by lemrey — APPROVED** [review]

**Pull request merged**